src/besselj.jl

```diff
@@ -151,41 +151,72 @@ function besselj1(x::Float32)
-    s12 = (-29393μ^12 + 305569628μ^11 - 569135407698μ^10 + 441734362333708μ^9 - 183408097342762543μ^8 + 45038004674954790968μ^7 - 6784356890740220944060μ^6 + 626292216963766324300664μ^5 - 34303895772550410316410943μ^4 + 1039206680182498322210164588μ^3 - 15233583205322702897380357650μ^2 + 81699279492428006742420030332μ - 67471218624230362526181277601) / 70368744177664
-
-
-    αp = s0 + s1/x^2 + s2/x^4 + s3/x^6 + s4/x^8 + s5/x^10 + s6/x^12 + s7/x^14 + s8/x^16 + s9/x^18 + s10/x^20 + s11/x^22 + s12/x^24
-    α = s0*x - s1/x - s2/3x^3 - s3/5x^5 - s4/7x^7 - s5/9x^9 - s6/11x^11 - s7/13x^13 - s8/15x^15 - s9/17x^17 - s10/19x^19 - s11/21x^21 - s12/23x^23
-
+    s2 = evalpoly(μ, (-25, 26, -1)) / 128
+    s3 = evalpoly(μ, (1073, -1187, 115, -1)) / 1024
+    s4 = evalpoly(μ, (-375733, 430436, -56238, 1540, -5)) / 32768
+    s5 = evalpoly(μ, (55384775, -64709091, 9716998, -397190, 4515, -7)) / 262144
+    s6 = evalpoly(μ, (-24713030909, 29215626566, -4733751627, 235370036, -4238531, 24486, -21)) / 4194304
+
+    if x < 15*v
+        s7 = evalpoly(μ, (7780757249041, -9268603618823, 1573356635461, -87480924739, 1989535379, -18939349, 63063, -33)) / 33554432
+        s8 = evalpoly(μ, (-26308967412122125, 31505470994964360, -5517359285625804, 329343318168440, -8593140373614, 106122595896, -598859404, 1252680, -429)) / 2147483648
+        s9 = evalpoly(μ, (14378802319925055947, -17285630127691126691, 3095397360215483916, -194026764558396188, 5536596631240042, -79979851361130, 597489288988, -2163210764, 3026595, -715)) / 17179869184
+        s10 = evalpoly(μ, (-19740662615375374580231, 23801928703130666089534, -4334421752432088249971, 281576004385356401192, -8556293060689145118, 137164402798287604, -1216961874423502, 5925778483368, -14587179035, 14318590, -2431)) / 274877906944
+        s11 = evalpoly(μ, (16629305448257355302267575, -20097626064642945009122253, 3708133715826433118900337, -247560495216085669748963, 7882145023568373727078, -136097258378143928354, 1355921534379626242, -7938052610367302, 26616800349043, -46575001473, 33302269, -4199)) / 2199023255552
+        s12 = evalpoly(μ, (-67471218624230362526181277601, 81699279492428006742420030332, -15233583205322702897380357650, 1039206680182498322210164588, -34303895772550410316410943, 626292216963766324300664, -6784356890740220944060, 45038004674954790968, -183408097342762543, 441734362333708, -569135407698, 305569628, -29393)) / 70368744177664
+        s13 = evalpoly(μ, (81121053093143918050102987600099,  -98383136980007985475571939027975, 18503851640550924693860888790250, -1284587641197372537980918233562, 43642612323123398148213493825, -832261743620394930038568309, 9606500048207046619311900, -69893489053616393418780, 325411350127146969525, -959952773579760385, 1717262634951770, -1676314996330, 692939975, -52003)) / 562949953421312
```

Comment:
these coeffs are being stored as `Int128` (they're big) which won't be good for performance. You probably want to fix that.